modules.d/01systemd-networkd/module-setup.sh

```diff
@@ -77,5 +77,6 @@ install() {
             "$systemdsystemconfdir"/systemd-networkd-wait-online.service \
             "$systemdsystemconfdir/systemd-networkd-wait-online.service/*.conf" \
             "$sysusersconfdir"/systemd-network.conf
+            "${systemdsystemconfdir}/network/*.link"
```

Comment:
The path `${systemdsystemconfdir}/network/` is wrong, already fixed upstream:
- https://github.com/dracutdevs/dracut/commit/744c6de5
- https://github.com/dracutdevs/dracut/commit/2eb733cc

The local `*.link` files are already added in line 70, so this line is not necessary:

https://github.com/openSUSE/dracut/blob/dcb9bdbf70650051957e2b93863ab14aab87c47e/modules.d/01systemd-networkd/module-setup.sh#L70

Reply:
Indeed, thanks for the hint, fixed.